samples/light_switch/README.rst

```diff
@@ -222,13 +216,21 @@ User interface
 .. note::
     If the brightness level is at the minimum level, you may not notice the effect of turning on the light bulb.
 
-..
-  FOTA behavior assignments
-  =========================
+FOTA behavior assignments
+=========================
 
-  LED 1:
-      Indicates the OTA activity.
-      Used only if the FOTA support is enabled.
+.. tabs::
+
+   .. group-tab:: nRF54L15 DK
+
+      LED 1:
+         Indicates the OTA activity.
+         Used only if the FOTA support is enabled.
```

Comment:
```suggestion
         Used only if the FOTA support is enabled.
         
```

Reply:
Plus can we add a line break before line 197(191), before the other device's tab?

Reply:
Fixed a few more in the same situation.